### PR TITLE
COOPR-564: Coopr CLI run scripts

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by nitinmotgi** [comment]
Lgtm :+1

**Comment by albertshau** [comment]
@wolf31o2 can you take a look at the scripts?

File: coopr-cli/bin/coopr-cli.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# limitations under the License.
+#
+
+# Attempt to set APP_HOME
+# Resolve links: $0 may be a link
+PRG="$0"
+bin=`dirname "${BASH_SOURCE-$0}"`
+bin=`cd "$bin"; pwd`
+lib="$bin"/../lib
+script=`basename $0`
```
Comment (wolf31o2) [inline coopr-cli/bin/coopr-cli.sh]: Several updates to this section...

```
PRG=${0} # Use this syntax, everywhere
bin=`dirname ${BASH_SOURCE-$0}` # No need for redundant quotes
bin=`cd ${bin}; pwd` # What do you do in the case of an error?
lib=`dirname ${bin}/lib` # No need for ../
script=`basename ${0}`
```

File: coopr-cli/bin/coopr-cli.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# Need this for relative symlinks.
+while [ -h "$PRG" ] ; do
+    ls=`ls -ld "$PRG"`
+    link=`expr "$ls" : '.*-> \(.*\)$'`
+    if expr "$link" : '/.*' > /dev/null; then
+        PRG="$link"
+    else
+        PRG=`dirname "$PRG"`"/$link"
+    fi
+done
```
Comment (wolf31o2) [inline coopr-cli/bin/coopr-cli.sh]: Actually, why is this even here? We don't use `PRG` anywhere.

File: coopr-cli/bin/coopr-cli.sh [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+if [ "$CLASSPATH" = "" ]; then
+  CLASSPATH=${lib}/coopr-cli-@@version@@.jar
+else
+  CLASSPATH=$CLASSPATH:${lib}/coopr-cli-@@version@@.jar
```
Comment (wolf31o2) [inline coopr-cli/bin/coopr-cli.sh:28]: Avoid using `@@version@@` and we don't need Maven to munge our script.

File: coopr-cli/bin/coopr-cli.sh [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+fi
+echo "${CLASSPATH}"
+echo "$script"
+java -cp ${CLASSPATH} -Dscript=$script co.cask.coopr.shell.CLIMain "$@"
```
Comment (wolf31o2) [inline coopr-cli/bin/coopr-cli.sh]: Don't convert positional parameters. Use `${@}` and remove the quotes.

File: coopr-cli/bin/coopr-cli.sh (hunk 1 above)
Comment (ghost) [inline coopr-cli/bin/coopr-cli.sh]: This not working correctly

File: coopr-cli/bin/coopr-cli.sh (hunk 2 above)
Comment (ghost) [inline coopr-cli/bin/coopr-cli.sh]: yes

File: coopr-cli/bin/coopr-cli.sh (hunk 3 above)
Comment (ghost) [inline coopr-cli/bin/coopr-cli.sh:28]: This consistent with cdap-cli

File: coopr-cli/bin/coopr-cli.sh (hunk 4 above)
Comment (ghost) [inline coopr-cli/bin/coopr-cli.sh]: ok

File: coopr-cli/bin/coopr-cli.sh (hunk 1 above)
Comment (ghost) [inline coopr-cli/bin/coopr-cli.sh]: I made some changes to this

**Comment by wolf31o2** [comment]
:+1: LGTM... :smile:

**Pull request merged**